src/db/index.js

```diff
@@ -744,7 +749,7 @@ module.exports = function(formio) {
         await unlock();
         return db;
       }
-      catch (err) {
+      catch (ignoreErr) {
```

Comment:
You probably didn't want to rename that one

Reply:
actually that was intentional - if an error happens with the database, there will be no lock to unlock, and `await unlock()` will throw an error - we want to surface the _original_ error, not the unlock error (so we can safely ignore the unlock error in other words)

Reply:
in other words, we want to throw the original `err` and not shadow that variable with the unlock error